### DISCUSSION TITLE
Can not build project.

### DISCUSSION THREAD

**Discussion by PlanCZero** [Dotnet-Boxed/Templates#50] [Q&A]
Hi. I'm using vs 2015 and create project from mvc6 boilerplate, After restore package (Reference,Dependencies), I try to build project but it always has error "Predefined type 'System.Object' is not defined or imported". Please advise. 

Regards.

**Comment by RehanSaeed** [comment]
It sounds like your NuGet dependencies have not been restored. Right click on references and hit restore. Also try deleting your project.lock.json file and then doing a restore.

Also have you updated to RC1 Update 1 and then run the `dnvm upgrade` command in PowerShell? There is a link in the pre-requisites section of the ReadMe on how to get everything installed.

**Comment by PlanCZero** [comment]
Hi. I do following your suggestion but still get error.

Severity Code Description Project File Line Error CS0518 Predefined type 'System.Object' is not defined or imported BPx1.DNX Core 5.0   

Regards.

**Comment by RehanSaeed** [comment]
Can you give me more details:
- Visual Studio Version (Is it Update 1?).
- DNVM Version.
- What are the steps you followed to reproduce the error.
- Are all the reference assemblies all restored? Do they have yellow exclamation marks on them?

**Comment by PlanCZero** [comment]
Hi. 
I just update my visual studio community 2015 to update 1 and create project with template "ASP.NET MVC 6 (RC1 Update 1) Boilerplate (Also RC1)" then restore references and dependencies. I also do dnvm upgrade on powershell too but It still have error when I build it. references and dependencies both success to restore.

![image](https://cloud.githubusercontent.com/assets/5127421/11889369/19b63b10-a57b-11e5-9d1b-9d017a5557e6.png)

**Comment by RehanSaeed** [comment]
Can you please send the output of the `dnvm version` and `dnvm list` command. Also a copy of your `project.json` file.

I'm guessing that there is something wrong with your installation, I've tested the project template on two machines just in case and it seems fine.

**Comment by PlanCZero** [comment]
Hello. Please find my dnvm version and list below.

![image](https://cloud.githubusercontent.com/assets/5127421/11896874/cf5a13a6-a5bc-11e5-97ca-e02a54307515.png)

Here is my project.json

{
  "userSecretsId": "BPy-31fae7bb-7b98-41ab-b791-7b0df1edda42",
  "version": "1.0.0-*",

  "dependencies": {
    "Boilerplate.Web.Mvc6": "2.0.0-rc1-final-3",
    "Boilerplate.Web.Mvc6.TagHelpers": "2.0.0-rc1-final-3",
    "Glimpse": "2.0.0-beta1",
    "Microsoft.AspNet.CookiePolicy": "1.0.0-rc1-final",
    "Microsoft.AspNet.Diagnostics": "1.0.0-rc1-final",
    "Microsoft.AspNet.IISPlatformHandler": "1.0.0-rc1-final",
    "Microsoft.AspNet.Mvc": "6.0.0-rc1-final",
    "Microsoft.AspNet.Mvc.TagHelpers": "6.0.0-rc1-final",
    "Microsoft.AspNet.Server.Kestrel": "1.0.0-rc1-final",
    "Microsoft.AspNet.StaticFiles": "1.0.0-rc1-final",
    "Microsoft.AspNet.Tooling.Razor": "1.0.0-rc1-final",
    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0-rc1-final",
    "Microsoft.Extensions.Configuration.Json": "1.0.0-rc1-final",
    "Microsoft.Extensions.Configuration.UserSecrets": "1.0.0-rc1-final",
    "Microsoft.Extensions.Logging": "1.0.0-rc1-final",
    "Microsoft.Extensions.Logging.Console": "1.0.0-rc1-final",
    "Microsoft.Extensions.Logging.Debug": "1.0.0-rc1-final",
    "Microsoft.VisualStudio.Web.BrowserLink.Loader": "14.0.0-rc1-final",
    "Newtonsoft.Json": "7.0.1"
  },

  "commands": {
    "web": "Microsoft.AspNet.Server.Kestrel"
  },

  "compilationOptions": {
    "emitEntryPoint": true
  },

  "frameworks": {
    "dnx451": {
      "frameworkAssemblies": {
        "System.Net.Http": "4.0.0.0",
        "System.ServiceModel": "4.0.0.0"
      }
    },
    "dnxcore50": {
      "dependencies": {
        "System.IO.Compression": "4.1.0-beta-23516",
        "System.Net.Http": "4.0.1-beta-23516"
      }
    }
  },

  "exclude": [
    "wwwroot",
    "node_modules",
    "bower_components"
  ],
  "publishExclude": [
    "node_modules",
    "bower_components",
    "**.xproj",
    "**.user",
    "**.vspscc"
  ],
  "scripts": {
    "prepublish": [ "npm install", "bower install", "gulp build" ]
  }
}

Regards

**Comment by RehanSaeed** [comment]
You have an old version of DNVM. See my version:

![untitled](https://cloud.githubusercontent.com/assets/2270364/11897015/a277c538-a583-11e5-979a-01a982210034.png)

You need to install the ASP.NET 5 RC1 Update 1 tools from the https://get.asp.net/.
